cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/PreferencesHttpHandler.java

```diff
@@ -74,6 +85,150 @@
     this.nsStore = nsStore;
   }
 
+  /**
+   * Returns user repository information
+   */
+  @Path("git/repo/info")
```

Comment:
repos/git/{repo}

repo is the repo name.

You can reference https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ConnectionHandler.java#L124

Reply:
so we should have two GET api

one is like "repos/git/" which is to list all the repos
the other is like "repos/git/{repo}" which is to get the specific "repo", where "repo" is the nickname /id of the repo